extensions/wikia/Listusers/SpecialListusers_ajax.php

```diff
@@ -4,30 +4,35 @@
  * @package MediaWiki
  * @subpackage SpecialPage
  * @author Piotr Molski <moli@wikia.com>
- * @version: $Id$
  */
-
 class ListusersAjax {
 
 	/**
 	 * ajax response for request params
 	 *
 	 * @author      Piotr Molski <moli@wikia-inc.com>
 	 * @version     1.0.0
-	 * @param       Array   $list
+	 *
+	 * @return AjaxResponse
 	 */
 	public static function axShowUsers ( ) {
-		global $wgRequest, $wgUser, $wgCityId;
-
+		global $wgUser, $wgCityId;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app